common/meta/grpc/src/grpc_client.rs

```diff
@@ -87,7 +101,9 @@ pub struct MetaGrpcClient {
 const AUTH_TOKEN_KEY: &str = "auth-token-bin";
 
 impl MetaGrpcClient {
-    pub async fn try_new(conf: &MetaGrpcClientConf) -> Result<MetaGrpcClient> {
+    pub async fn try_new(
+        conf: &MetaGrpcClientConf,
+    ) -> std::result::Result<MetaGrpcClient, Infallible> {
```

Comment:
Why not just rename `try_new` to `new` and change the return type from `Result<T, Infallible>` to `T` if a method does not return any error.

Reply:
I'm not very sure about whether in the future an error will return.
IMHO, it is very likely to be.